content/webapp/types/articles.ts

```diff
@@ -28,6 +28,7 @@ export type ArticleBasic = {
 export type Article = GenericContentFields & {
   type: 'articles';
   format?: Format<ArticleFormatId>;
+  readingTime?: number | null;
```

Comment:
This seems like a bit of an odd type; it becomes `number | undefined | null` – could we simplify it to `readingTime | undefined`?

Reply:
> I think we could do it once, when we compute the readingTime, then let the downstream components just react to whether it's `null`. That means you only need to write that logic once.

Finally addressed this properly I think, along with the few other things :)